content/posts/2024-04-01-toil.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Toil"
+date: "2024-04-01"
+tags: "note, life"
+---
+
+After cheating death himself, [Sisyphus](https://en.m.wikipedia.org/wiki/Sisyphus) was punished to push a large boulder up a steel hill for eternity. Every time he reached the top, the boulder would roll back down and Sisyphus had to start over.
+
+Our life is full of such toil. Mowing the lawn, cleaning the dishes and doing our taxes are examples of toil that prevents us from doing sustainable work. We should always seek to minimize toil where possible. We have dishwashers and spam filters to help us fight toil, but there are some reoccurring tasks that cannot be automated.
+
+Sisyphus eventually found peace with this fate. Instead of seeing his punishment as an eternal burden, he saw it as his life's purpose. Maybe those comparatively small annoyances in our life are worth enjoying. They're part of what makes us human.
```

Comment:
```suggestion
Sisyphus eventually found peace with his fate. Instead of seeing his punishment as an eternal burden, he saw it as his life's purpose. Maybe those comparatively small annoyances in our life are worth enjoying. They're part of what makes us human.
```